pdp/handlers.go

```diff
@@ -909,15 +914,213 @@ func (p *PDPService) handleAddRootToProofSet(w http.ResponseWriter, r *http.Requ
 		return true, nil
 	}, harmonydb.OptionRetry())
 	if err != nil {
-		log.Errorw("Failed to insert into database", "error", err, "txHash", txHash.Hex(), "subroots", subrootInfoMap)
+		log.Errorw("Failed to insert into database", "error", err, "txHash", txHashLower, "subroots", subrootInfoMap)
 		http.Error(w, "Internal server error", http.StatusInternalServerError)
 		return
 	}
 
 	// Step 10: Respond with 201 Created
+	w.Header().Set("Location", path.Join("/pdp/proof-sets", proofSetIDStr, "roots/added", txHashLower))
 	w.WriteHeader(http.StatusCreated)
 }
 
+// handleGetRootAdditionStatus handles GET /pdp/proof-sets/{proofSetID}/roots/added/{txHash}
+func (p *PDPService) handleGetRootAdditionStatus(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	// Step 1: Verify that the request is authorized using ECDSA JWT
```

Comment:
Comment misleading since in general its not JWT and its hardcoded to not be JWT right now

Reply:
Yeah, but I'm just following the pattern in the file that exists now:

```
$ grep 'Verify that the request' pdp/handlers.go 
        // Verify that the request is authorized using ECDSA JWT
        // Step 1: Verify that the request is authorized using ECDSA JWT
        // Step 1: Verify that the request is authorized using ECDSA JWT
        // Step 1: Verify that the request is authorized using ECDSA JWT
        // Step 1: Verify that the request is authorized using ECDSA JWT
        // Step 1: Verify that the request is authorized using ECDSA JWT
        // Step 1: Verify that the request is authorized using ECDSA JWT
        // Step 1: Verify that the request is authorized using ECDSA JWT
```

Do you want me to remove all of those, or just these new ones?